crates/biome_js_analyze/src/lint/nursery/use_explicit_function_return_type.rs

```diff
@@ -110,6 +121,15 @@ declare_lint_rule! {
     /// (() => {})();
     /// ```
     ///
+    /// ```ts
```

Comment:
It is good to add examples to demonstrate the exceptions to the rule. However, I think we should also describe the exceptions to the rule in the description (before `## Examples`). We should also document the previous exceptions to the rule introduced in your previous PRs.